webdev/static/azion.json

```diff
@@ -0,0 +1,20 @@
+{
+  "name": "__DEFAULT__",
+  "env": "production",
+  "type": "static",
+  "version-id": "20230511071808",
```

Comment:
I think we can remove this version-id, leave it blank

Reply:
I agree